requirements.txt

```diff
@@ -0,0 +1,9 @@
+numpy~=1.21.2
```

Comment:
I think version number can be removed for most packages, just use `numpy`, same for others.